examples/src/main/java/com/opengamma/strata/examples/finance/CalibrationCheckExample.java

```diff
@@ -203,7 +208,7 @@ public static void performance_calibration_pricing() {
       for (CurveNode node : nodes) {
         if (!(node instanceof IborFixingDepositCurveNode)) {
           // IborFixingDeposit is not a real trade, so there is no appropriate comparison
-          trades.add(node.trade(VALUATION_DATE, ObservableValues.ofIdMap(quotes)));
+          trades.add(node.trade(VALUATION_DATE, MarketData.builder().addValues(quotesByKey).build()));
```

Comment:
Yes, my thought was to use `ObservableId` here, maybe naming the method as such. Perhaps as ids and keys develop we may be able to improve, but I don't want to lose this method relative to what we had with `ObservableValues`